Tests/GDSAnalyticsTests/Screens/ScreenViewTests.swift

```diff
@@ -7,24 +7,18 @@ final class ScreenViewTests: XCTestCase {
                               titleKey: "Welcome to this app")
         
         XCTAssertEqual(view.screen, MockScreen.welcome)
-        XCTAssertEqual(view.title, "Welcome to this app")
+        XCTAssertEqual(view.title, "welcome to this app")
     }
     
     func testParameters() {
-        let view = ScreenView(screen: MockScreen.welcome,
-                              titleKey: "Welcome to this app")
-        
-        XCTAssertEqual(view.parameters, [
-            "title": "welcome to this app"
-        ])
-    }
-    
-    func testParameterTruncation() {
```

Comment:
do we no longer have a test for truncation of the title?
please can we add that back in.